src/coresyf_image_crop/coresyf_image_crop.py

```diff
@@ -1,32 +1,31 @@
 #!/usr/bin/python2
-import zipfile
 import sys
 import os
-from osgeo import ogr
+from osgeo import ogr, osr
 from coresyftools.tool import CoReSyFTool
 from sridentify import Sridentify
 
 
-def read_zip_shapefile(filepath):
+def read_shapefile(folder_path):
     '''
-    It opens a zip file containing a shapefile and returns an handle to the
+    It opens folder with a shapefile and returns an handle to the
     OGRDataSource (a GDAL OGR object with the shapefile data).
     '''
     # Get main file of the shapefile
-    input_list = [os.path.join(filepath, x) for x in os.listdir(filepath)
+    input_list = [os.path.join(folder_path, x) for x in os.listdir(folder_path)
                   if x.endswith(".shp")]
     if not input_list:
-        sys.exit("Shapefile not found in '%s'!" % filepath)
+        sys.exit("Shapefile not found in '%s'!" % folder_path)
     shapefile_shp = input_list[0]
     try:
         driver = ogr.GetDriverByName('ESRI Shapefile')
         data_source = driver.Open(shapefile_shp, 0)  # 0 means read-only.
         if not data_source:
             raise IOError()
     except IOError:
-        sys.exit("Error. Unable to open shapefile '%s'!" % filepath)
+        sys.exit("Error. Unable to open shapefile '%s'!" % folder_path)
     except Exception:
-        sys.exit("Unexpected error when opening shapefile '%s'!" % filepath)
+        sys.exit("Unexpected error when opening shapefile '%s'!" % folder_path)
```

Comment:
If we are in presence of a failure we aren't able to handle why not just letting it pass to the caller? 
Otherwise, we will have every function polluted with the same `except Exception` block, which could be placed just in the main script code.